robots/flakefinder/main.go

```diff
@@ -172,3 +179,14 @@ func main() {
 	}
 
 }
+
+func BuildReportOutputPath(o Options) string {
+	var outputPath = ReportsPath
+	if o.IsPreview {
+		outputPath = strings.Join([]string{outputPath, "preview"}, "/")
+	}
+	if o.ReportOutputChildPath != "" {
```

Comment:
Can you use `filepath.Join`? No need to check if the string is empty.

Reply:
Nice hint, thanks!